samples/graph-activity-feed/csharp/ActivityFeedNotification/Helpers/TeamMembersNotificationRecipient.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Newtonsoft.Json;
+
+namespace Microsoft.Graph
+{
+    //
+    // Summary:
+    //     The type TeamMembersNotificationRecipient.
```

Comment:
This summary alignment doesn't looks correct.

Reply:
Done